### PR TITLE
Feature/netstats styles

### PR DESCRIPTION
**Pull request by adamking**
Updates styles for netstats per #501

### CODE REVIEW

File: netstats/src/views/layout.jade [hunk 1]
```diff
@@ -3,34 +3,55 @@ doctype html
 html(ng-app="netStatsApp")
   head
     meta(name="viewport", content="width=device-width, initial-scale=1.0, maximum-scale=1.0")
-    title Kowala Test Network
+    title Kowala - kUSD Test Network
     style(type="text/css") [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak { display: none !important; }
-    link(rel='stylesheet', href='//fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600"')
-    link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css')
+    link(rel="stylesheet", href="https://fonts.googleapis.com/css?family=Poppins:300,400,500")
+    link(rel="stylesheet", href="https://cdn.kowala.tech/styles/kowala.css")
```
Comment (yourheropaul) [inline netstats/src/views/layout.jade]: This is always using the main public CDN. Is there a way to make it use the dev CDN for dev and the prod CDN for prod?

File: netstats/src/views/layout.jade [hunk 2]
```diff
@@ -3,34 +3,55 @@ doctype html
...
-              a(href="https://github.com/kowala-tech/kcoin", target="_blank") Github
-            li
-              a(href="https://docs.kowala.tech", target="_blank") Documentation
-            li
-              a(href="https://t.me/kowalaofficial", target="_blank") Telegram Community
-    section#content.container-fluid
-      block content
+  body.light-blue-background.light-#fff-text
+    #mobile-menu(data-reveal="")
+      img(src="https://cdn.kowala.tech/images/kowala/kowala-white.svg" title="Kowala")
```
Comment (yourheropaul) [inline netstats/src/views/layout.jade]: This is also the prod CDN

File: netstats/src/views/layout.jade [hunk 3]
```diff
@@ -3,34 +3,55 @@ doctype html
...
+      ul
+        li
+          a(href="https://explorer.zygote.kowala.tech" rel="noopener" target="_blank") BLOCK EXPLORER
```
Comment (yourheropaul) [inline netstats/src/views/layout.jade]: There will soon be prod versions of these links. Can we link dev -> dev and prod -> prod?

File: netstats/src/views/layout.jade (hunk 1 above)
Comment (adamking) [reply netstats/src/views/layout.jade]: Updated

File: netstats/src/views/layout.jade (hunk 2 above)
Comment (adamking) [reply netstats/src/views/layout.jade]: Updated

File: netstats/src/views/layout.jade (hunk 3 above)
Comment (adamking) [reply netstats/src/views/layout.jade]: Updated and added social media meta

**Comment by adamking** [comment]
@yourheropaul I've updated it for both envs and added some meta for social media.

**Pull request merged**